build/validateNodeVersion.js

```diff
@@ -0,0 +1,105 @@
+const axios = require('axios');
+
+const SUPPORTED_LTS = "Carbon";
+const VERSION = process.versions.node;
+const NODE_JS_VERSION_LIST_URL="https://nodejs.org/dist/index.json"
+const MIN_MAJOR_SUPPORT = 8;
+const MIN_MINOR_SUPPORT = 9;
+const MIN_PATCH_SUPPORT = 4;
+
+function fail() {
+    console.log(false);
+}
+
+function pass() {
+    console.log(true);
+}
+
+/**
+ * Returns true if greater or same version as minimum supported version
+ *
+ * @param {Number} major - Major version number
+ * @param {Number} minor - Minor version number
+ * @param {Number} patch - Patch version number
+ * @returns {boolean} True if a supported LTS version of NodeJS
+ */
+function compareVersion(major, minor, patch) {
+    // If major version is greater, it's valid
+    if (major > MIN_MAJOR_SUPPORT) {
```

Comment:
Can this be a problem, lets say someone is using node 10? There could be breaking changes

Reply:
`compareVersion()` only compares against our minimum requirements of node `8.9.4`. After this is executed, we do a check against LTS versions of node, which are supported (at least) by conventional-github-releaser.